packages/checkout/widgets-lib/src/views/top-up/TopUpView.tsx

```diff
@@ -27,25 +31,30 @@ import {
   getBridgeFeeEstimation,
   getOnRampFeeEstimation,
 } from '../../lib/feeEstimation';
-import { CryptoFiatActions, CryptoFiatContext } from '../../context/crypto-fiat-context/CryptoFiatContext';
+import {
+  CryptoFiatActions,
+  CryptoFiatContext,
+} from '../../context/crypto-fiat-context/CryptoFiatContext';
 import { OnRampWidgetViews } from '../../context/view-context/OnRampViewContextTypes';
 import { EventTargetContext } from '../../context/event-target-context/EventTargetContext';
 import { TopUpMenuItem } from './TopUpMenuItem';
 
 interface TopUpViewProps {
-  widgetEvent: IMTBLWidgetEvents,
-  checkout?: Checkout,
-  provider?: Web3Provider,
-  showOnrampOption: boolean,
-  showSwapOption: boolean,
-  showBridgeOption: boolean,
-  tokenAddress?: string,
-  amount?: string,
+  widgetEvent: IMTBLWidgetEvents;
+  checkout?: Checkout;
+  provider?: Web3Provider;
+  showOnrampOption: boolean;
+  showSwapOption: boolean;
+  showBridgeOption: boolean;
+  tokenAddress?: string;
+  amount?: string;
   analytics: {
-    userJourney: UserJourney
-  },
-  onCloseButtonClick: () => void,
-  onBackButtonClick?: () => void,
+    userJourney: UserJourney;
+  };
+  onCloseButtonClick: () => void;
+  onBackButtonClick?: () => void;
+  heading?: [key: string, options?: $Dictionary];
```

Comment:
@dreamoftrees 
after testing this, I noticed that if we pass an already translated string it wont re-render if you set a different language

So I opted to pass an array and use give the params to the translation function. Not adding the `defaultValue` as for this use case this component has the default copy.